### PR TITLE
airflow and dataflow script 

### PR DESCRIPTION
**Pull request by Sihan-Mao**
gcstobg operators in qalert_airflow used write_disposition == "write_empty". I put it this way coz there is no table before. Maybe change it to "write_append" afterwards?

### CODE REVIEW

**Comment by Sihan-Mao** [comment]
@jtotoole

File: airflow_scripts/dags/qalert_airflow.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+
+qalert_requests_bq = GoogleCloudStorageToBigQueryOperator(
+    task_id='qalert_requests_bq',
+    destination_project_dataset_table='{}:311.requests'.format(os.environ['GCP_PROJECT']),
+    bucket='{}_311'.format(os.environ['GCS_PREFIX']),
+    source_objects=["requests/avro_output/{}/{}/{}/*.avro".format(dt.strftime('%Y'),
+                                                         dt.strftime('%m').lower(),
+                                                         dt.strftime("%Y-%m-%d"))],
+    write_disposition='WRITE_EMPTY',
```
Comment (jtotoole) [inline airflow_scripts/dags/qalert_airflow.py]: You want `WRITE_APPEND` here; that will create a table if needed but add to an existing table if not. `WRITE_EMPTY` will only write to empty tables, so it doesn't work for recurring jobs.

**Comment by jtotoole** [comment]
This looks good! Just make that `WRITE_APPEND` change and we can merge. 

We'll want to refactor the airflow script to add an additional step for reverse-geocoding the `requests` table and maybe also joining it with activities to get close dates where applicable. We can do that later, however; I'm working on a geocoding helper function to use in the firearms script that we can also use here.

**Comment by jtotoole** [comment]
looks good! i'm gonna merge. you can then check out `master` and do `git pull origin master` to get these latest changes on your `master` branch.

**Pull request merged**